integrations/elasticsearch/config.json

```diff
@@ -0,0 +1,53 @@
+{
+    "display_name": "Elasticsearch",
+    "integration_type": "event_output",
+    "supported_event_types": [],
+    "required_fields": [],
+    "max_send_retries": 5,
+    "polling_enabled": false,
+    "test_connection_enabled": false,
+    "parameters": [
+         {
+            "value": "url_address",
+            "label": "URL address",
+            "type": "string",
+            "required": true,
+            "help_text": "The URL address of your elasticsearch instance, i.e. http[s]://1.2.3.4 or http[s]://elastic.local."
+        },
+        {
+            "value": "skip_cert_validation",
```

Comment:
Let's keep things positive!
Change to `verify` With label 'Verify SSL server certificate' and make it default Ture